src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,15 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <UserInfo user={post.user} />
```

Comment:
A user can be null, better to add the check here
<img width="496" alt="image" src="https://github.com/user-attachments/assets/82f5c47f-c7f8-48fe-b41e-b26874ca6f49">
```suggestion
      {post.user && <UserInfo user={post.user} />}
```